routes/api/v1/dashboards.js

```diff
@@ -0,0 +1,69 @@
+const Dashboard = require('../../../models/dashboards.js')
+const { Widget } = require('../../../models/widgets.js')
+
+const express = require('express')
+const router = express.Router()
+
+router.get('/', (request, response, next) => {
+  Dashboard.find().exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.post('/', (request, response, next) => {
+  Dashboard.create( request.body )
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.get('/:id', (request, response, next) => {
+  Dashboard.findById( request.params.id ).exec()
+    .then( dashboard => response.json( dashboard ) )
+    .catch( error => next( error ) )
+})
+
+router.put('/:id', (request, response, next) => {
+  Dashboard.findByIdAndUpdate( request.params.id, request.body ).exec()
```

Comment:
This information does not get stored in this record. If the write in a key that doesn't exist in the schema for dashboard, we don't get an error message, but we also don't see that key or its value in the record.